pymemcache/test/test_client.py

```diff
@@ -71,6 +72,24 @@ def setsockopt(self, level, option, value):
         self.socket_options.append((level, option, value))
 
 
+class MockUnixSocket(object):
+
+    def __init__(self, socket_path):
+        if os.path.exists(socket_path):
+            os.remove(socket_path)  # Make sure no stalling connection exists
```

Comment:
"stalled connection" might be clearer wording?

Reply:
Hi, sorry for the ghosting. Been pretty busy on other projects. All your statements make sense and sorry for the auto-formatting (I forgot to disable it).
I'll try to find some time to apply the modifications you suggested.

Cheers,